gallery/src/GalleryNav/GalleryNavIcons.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+export const GalleryNavIconCSS3 = () => (
+  <svg className="gallery-nav-icon gallery-nav-icon--css3"
+       width="20"
+       height="20"
```

Comment:
I'd leave the actual sizing up to CSS.  And it might be cleanest if we try to stay close to what FA does: setting the height to `1em`

Reply:
> I'd leave the actual sizing up to CSS. And it might be cleanest if we try to stay close to what FA does: setting the height to `1em`

`1.2em` looked a better match for the SASS logo below. `1em` it was too small and hard to read. I also tweaked the vertical spacing by `0.1`

https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/c438b43bdd09ecd4b3202e00e519494b7874d9d2

Reply:
> 1.2em looked a better match for the SASS logo below. 1em it was too small and hard to read. I also tweaked the vertical spacing by 0.1

Consider these changes: 900d64c9 .  IMO it's valuable to stick to the `1em` height because having inline elements of different heights next to each other can cause weirdness around vertical alignment and the height of the overall line box. Also note that the width needs to be `1.25em` to match FA full-width icons